rules/order.js

```diff
@@ -30,6 +55,7 @@ module.exports = {
         'box-sizing',
         'display',
         'overflow',
+        'vertical-align',
```

Comment:
Aren't all stylistic rules inherently arbitrary, and don't they all cause back-and-forth with the linter for people who don't know them? Are we arguing that we should never enforce any consistent styling anywhere?